apps/hash-integration-worker/src/main.ts

```diff
@@ -66,11 +69,25 @@ async function run() {
     port: parseInt(getRequiredEnv("HASH_GRAPH_API_PORT"), 10),
   });
 
+  const vaultClient = createVaultClient();
+  if (!vaultClient) {
+    throw new Error("Vault client not created");
+  }
```

Comment:
**Blocking:** This works when running the integration worker locally, but it doesn't work when running the integration worker in docker. I assume this is because we aren't passing the correct vault environment variables to the container.